ts_src/psbt.ts

```diff
@@ -117,8 +117,7 @@ export class Psbt {
       __NON_WITNESS_UTXO_BUF_CACHE: [],
       __TX_IN_CACHE: {},
       __TX: (this.data.globalMap.unsignedTx as PsbtTransaction).tx,
-      // Old TransactionBuilder behavior was to not confirm input values
-      // before signing. Even though we highly encourage people to get
+      // Even though we highly encourage people to get
```

Comment:
I don't think these comments should be removed. Perhaps changed.

ie. TransactionBuilder -> "Psbt's previously removed predecessor" etc.

Reply:
- updated comments to include the reference to `TransactionBuilder`